src/app/_components/Form/FormFileInput/FormFileInput.tsx

```diff
@@ -37,26 +37,21 @@ export function FormFileInput({
   accept,
   ...rest
 }: FormFileInputProps) {
-  const loadedFile = files?.[0]
-
   function resetFiles() {
```

Comment:
We are working with a single file now, and yet, we have the functions `resetFiles` and `loadFiles`? Seems confusing?